Multisig/UI/App/SceneDelegate.swift

```diff
@@ -11,18 +11,18 @@ import SwiftUI
 
 class SceneDelegate: UIResponder, UIWindowSceneDelegate {
     var window: UIWindow?
-
+        
     func scene(_ scene: UIScene, willConnectTo session: UISceneSession, options connectionOptions: UIScene.ConnectionOptions) {
         // Use this method to optionally configure and attach the UIWindow `window` to the provided UIWindowScene `scene`.
         // If using a storyboard, the `window` property will automatically be initialized and attached to the scene.
         // This delegate does not imply the connecting scene or session are new (see `application:configurationForConnectingSceneSession` instead).
 
         // Get the managed object context from the shared persistent container.
         let context = CoreDataStack.shared.persistentContainer.viewContext
-
+        let viewState = ViewState.shared
```

Comment:
I would remove the environment object all together, because it doesn't work for our use cases. Please see another comment in the ContentVIew